### DISCUSSION TITLE
2022-01-26 Weekly Meeting 40 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#169] [Weekly Meetings]
Weekly Meeting 40 is happening on Wednesday January 26th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- working groups check in: 
  - [website group](https://github.com/omigroup/website)
  - [gLTF working group](https://github.com/omigroup/gltf-extensions)
  - [WASM working group](https://github.com/omigroup/omi-wasm-group)

### Announcements and Updates

- What does future structure look like Master discussion
  - New Roles in OMI discussion
  - Discuss a hackathon as a group: #173 
  - Backlog refinement session recap: #172
 - Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
We had our first backlog refinement session: https://github.com/omigroup/omigroup/issues/172

**Comment by mrmetaverse** [comment]
Created a proposal on behalf of Evo to discuss a hackathon as a group: https://github.com/omigroup/omigroup/issues/173

**Comment by mrmetaverse** [comment]
Are you familiar with our milestones system? 
[![Screenshot_20220126-084531](https://user-images.githubusercontent.com/63426722/151174199-eea139e8-cd07-4751-9da4-bf97b58bdcee.png)](https://github.com/omigroup/omigroup/milestones)

**Comment by mrmetaverse** [comment]
No volunteers to record, so this session will be only notes.

**Comment by mrmetaverse** [comment]
Rick will be taking their usual notes and adding here, I will also do my best to add notes as per usual.

**Comment by mrmetaverse** [comment]
# RE: Website group 

Want to help out? We are seeking images of our community in the Metaverse. Selfies with your group / friends in the metaverse, etc. 
- add here: https://airtable.com/shrZAsV1MA5QJctYx

Website has been updated to match the figma design. Still need to update some things like hover-state. We are ready to go live, and iterate from there. Pull request to come. 

also this pull request: https://github.com/omigroup/website/pull/49

**Comment by mrmetaverse** [comment]
One other creative way of helping is to photoshop, or take a selfie of your different avatars into a single shot. Help people understand a need for interoperability by demonstrating visually the challenges of traversing across the metaverse as different avatars. If you want to submit a group photo, just make sure the background can be cut out if needed. White, or greenscreen, or clear bg is preferred.

**Comment by mrmetaverse** [comment]
# RE: gLTF working group 

- last weeks agenda is available here: https://github.com/omigroup/gltf-extensions/discussions/51 
- tomorrows agenda: https://github.com/omigroup/gltf-extensions/discussions/52 
- tomorrow group will review pull request for audio emitter which is pretty much ready to go 
- beginning new spec for colliders and basic physics in gLTF objects (collision boxes, areas for triggered events, specify objects as static bodies for physics or dynamic bodies, etc. Once together we can add basic physics and collision detection in gLTF scenes which will enhance worlds, items, avatars, etc.) Overlapping potentially with WASM group. 
- This group is always looking to grow and meets on Thursdays at 22:30 UTC

**Comment by mrmetaverse** [comment]
# RE: WASM Group 

Defining scope, and getting aligned around what Web Assembly does. 
Spoke recently about APIs and integrating with WASM. discussed programming languages, etc. 
No agenda last time but working on that. 

Alternating between sessions with an agenda, versus more of a show and tell style. (great example of different working agreements and meeting styles!) 

Next week Nova is presenting Stardust! 

Input messages, how high level we want to be, and discussing pain points in other engines with dynamic scripting. The week after we will have another structured talk with an agenda. It's loose for now, and still taking shape. If you want to help steer where we are taking this, now is probably the time. 

We have a github discussion where we are tracking [Use Cases](https://github.com/omigroup/omi-wasm-group/discussions/13)

Hoping to narrow or refine scope and draft early user stories to help define and track the work.

**Reply by mrmetaverse** [reply]
Oi! shoutout to the WASM group for suggesting working agreement: https://github.com/omigroup/omi-wasm-group/issues/1

**Comment by mrmetaverse** [comment]
# RE: Elections x Future structure
- next election is next week 2/2 
- We are not locked in, if we need to make changes to the OMI structure during the next term to the overall OMI structure, we can and will. We should not brush this conversation under the rug until next next election. 
- You can nominate chairs in [this new issue](https://github.com/omigroup/omigroup/issues/174) 
- Tracking all of this in this Milestone: https://github.com/omigroup/omigroup/milestone/2
- The new structure of OMI proposed discussion: https://github.com/omigroup/omigroup/discussions/157
- OMI responsibilities Audit: https://hackmd.io/@mrmetaverse/omi-tasks-audit 
- Potential New Roles as a result of the responsibilities audit: https://github.com/omigroup/omigroup/discussions/165 
- Review our **core values**, as they help define our structure: 
  - We collaborate, driven by research, privacy and accessibility, to make the metaverse more human.
  - We work incrementally, iteratively, and with intent.
  - We are creating a cooperative, sustainable culture of innovation.
  - We empower the people making the metaverse.
  - We must cultivate resiliency by championing diversity, and freedom of choice.

**Comment by mrmetaverse** [comment]
# RE: Potential future Hackathon 

Evo is AFK so I am doing my best to track notes [here](https://github.com/omigroup/omigroup/issues/173)

**Comment by mrmetaverse** [comment]
Rick took notes today: 

![image](https://user-images.githubusercontent.com/63426722/151220654-0e971d6e-953b-4f25-a856-dc82ecbc941c.png)
